child_sync_wp/wizards/staff_notification_settings.py

```diff
@@ -57,6 +56,21 @@ def set_sponsorship_it_id(self):
             str(self.sponsorship_it_id.id)
         )
 
+    @api.multi
+    def get_sponsorship_fr_id(self):
+        return self.env['ir.config_parameter']\
+            .get_param('child_wp.sponsorship_notify_fr_id', '18001')
```

Comment:
I can't open the general settings view in v11 because the user 18001 does not exist. Can't we just return None and avoid sending the notification when the param is not defined ?

Reply:
Also, the associated labels are missing in the settings view

Reply:
@quentingigon Yes, please remove the constant and fix the view

Reply:
@quentingigon you didn't fix this yet.